README.md

```diff
@@ -38,15 +38,16 @@ In the above, the class `Max[T]` signifies that the `+` operator should actually
 
 ## Documentation
 
-To learn more and find links to tutorials and information around the web, check out the [website](https://twitter.github.io/algebird).
+To learn more and find links to tutorials and information around the web, check out the [Algebird website](https://twitter.github.io/algebird).
 
-The latest API docs are hosted on Algebird's [ScalaDoc index](http://twitter.github.io/algebird/api/).
+The latest API docs are hosted on Algebird's [ScalaDoc index](https://twitter.github.io/algebird/api/).
 
 ## Get Involved + Code of Conduct
 
-Pull requests and bug reports are always welcome!
+Pull requests and bug reports are always welcome! Check out our [Contributing guide](https://twitter.github.io/algebird/contributing.html) for information on what we most need help with and how you can get started contributing.
+
+Discussion occurs primarily on the Algebird Gitter channel: [![Chat](https://badges.gitter.im/twitter/algebird.svg)](https://gitter.im/twitter/algebird?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Discussion occurs primarily on the [Algebird mailing list](https://groups.google.com/forum/#!forum/algebird).
```

Comment:
Should we remove the mailing list entirely? Seems like some might still prefer it.